src/plugins/kibana_react/public/page_template/no_data_page/no_data_page_body/no_data_page_body.tsx

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { EuiFlexGrid, EuiLink, EuiSpacer, EuiText, EuiTextColor } from '@elastic/eui';
+import React, { ReactElement } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { NoDataPageProps } from '../no_data_page';
+import { KibanaPageTemplateSolutionNavAvatar } from '../../solution_nav';
+
+type NoDataPageBodyProps = {
+  actionCards: ReactElement[];
+} & Omit<NoDataPageProps, 'actions'>;
+
+export const NoDataPageBody = (props: NoDataPageBodyProps) => {
```

Comment:
Ultimatively, this is all Analytics apps would need to render. A simple page with `noConfig` props passed to it.

Reply:
I think this is still too low level. They'll need to use the higher `NoDataPage` component at least to get any action cards to display. 

Though, even when I tried swapping out the `KibanaPageTemplate` for `NoDataPage`, we still have layout issues because the template is handling the centering and width constraints.